synapse/push/push_tools.py

```diff
@@ -46,7 +46,7 @@ async def get_badge_count(store: DataStore, user_id: str, group_by_room: bool) -
     return badge
 
 
-async def get_context_for_event(storage: Storage, state_handler, ev, user_id):
+async def get_context_for_event(storage: Storage, ev, user_id):
```

Comment:
What about type hints for the other args?

```suggestion
async def get_context_for_event(storage: Storage, ev: EventBase, user_id: str) -> Dict[str, str]:
```

I think>